### ISSUE TITLE
Display datetime64 arrays without showing local timezones

### ISSUE THREAD

**Issue by shoyer** [pydata/xarray#439]
NumPy has an unfortunate way of adding local timezone offsets when printing datetime64 arrays:
```
<xray.DataArray 'time' (time: 4000)>
array(['1999-12-31T16:00:00.000000000-0800',
       '2000-01-01T16:00:00.000000000-0800',
       '2000-01-02T16:00:00.000000000-0800', ...,
       '2010-12-10T16:00:00.000000000-0800',
       '2010-12-11T16:00:00.000000000-0800',
       '2010-12-12T16:00:00.000000000-0800'], dtype='datetime64[ns]')
Coordinates:
  * time       (time) datetime64[ns] 2000-01-01 2000-01-02 2000-01-03 ...
```

We should use custom formatting code to remove the local timezone (to encourage folks just to use naive timezones/UTC).

**Comment by andreas-h** [comment]
can you give a pointer to where this would need to be done in the code?

**Comment by shoyer** [comment]
https://github.com/xray/xray/blob/v0.5.2/xray/core/formatting.py#L250

The `repr` call on this line would need to be replaced with another function that checks the dtype of the numpy array and uses custom handling for datetime64.

**Comment by sjpfenninger** [comment]
Would it not make sense to use a pandas.DatetimeIndex instead of pure-numpy datetimes? It seems that DatetimeIndex already solves lots of the weirdness in the underlying numpy datetime objects and adds a bunch of useful functionality for things like groupby operations.

**Comment by andreas-h** [comment]
sounds good to me, @sjpfenninger, but I don't know the internals ...

**Comment by shoyer** [comment]
The good news is that I fixed this display bug upstream in [NumPy 1.11](https://github.com/numpy/numpy/blob/master/doc/release/1.11.0-notes.rst), which came out last month. If you upgrade to NumPy 1.11, datetimes are always treated as timezone naive. So I think we can actually consider this issue fixed.
